packages/terra-worklist-data-grid/src/subcomponents/Cell.module.scss

```diff
@@ -2,6 +2,11 @@
 @import '../orion-fusion-theme/Cell.module';
 @import '../clinical-lowlight-theme/Cell.module';
 
+@mixin terra-worklist-data-grid-selectable-hover {
+  background-color: var(--terra-worklist-data-grid-hover-background-color, #d8e5ed);
+  cursor: pointer;
+}
```

Comment:
Why is the mixin being defined in the Cell component if it is used in the Cell and Row components.  Shouldn't it be in its own separate file that can be imported?

Reply:
Changed